### ISSUE TITLE
dual (MonomialIdeal, RingElement) throws unexpected error

### ISSUE THREAD

**Issue by aarondall** [Macaulay2/M2#884]
An error is thrown when computing the Alexander dual of a monomial ideal with respect to a ring element.

```
i1 : S = kk[x];

i2 : I = monomialIdeal "x2"

                    2
o2 = monomialIdeal x
o2 : MonomialIdeal of S

i3 : dual (I, x^3)
stdio:3:1:(3): error: no method for adjacent objects:
--            null (of class Nothing)
--                            2
--    SPACE   (monomialIdeal x , {3}, OptionTable{Strategy => 0}) (of class Sequence)
```

The methods `dual (MonomialIdeal, List)` and `dual MonomialIdeal` work as expected.

```
i4 : dual (I, {3})

                    2
o4 = monomialIdeal x

o4 : MonomialIdeal of S

i5 : dual I

o5 = monomialIdeal x

o5 : MonomialIdeal of S
```

**Comment by mikestillman** [comment]
The function is null? I'll take a look at it. Should be easy to fix, probably.  I guess it isn't on the test path!

**Comment by mikestillman** [comment]
Yes, that function was not even defined, and it was being called!    I fixed it to call the correct function, instead of null...  And added a test

Fixed in commit 7728bb1fcc3f822b
